### ISSUE TITLE
Sessions Should Not Be Indefinite

### ISSUE THREAD

**Issue by Sephster** [WebPA/WebPA#49]
When logged in, there is no inactivity timeout. We should add this in to improve security in the platform.

**Comment by spvickers** [comment]
My understanding of the current code is that it intentionally keeps the session alive whilst WebPA is open in a browser window.  Otherwise the session will expire after a period of inactivity.

**Comment by Sephster** [comment]
Thanks @spvickers. I haven't dug into the underlying code yet. I listed this because I was working in WebPA last night and put my computer into standby. This morning, I was able to use the same browser window to continue working in WebPA without the need to login.

I would expect that if I have not interacted with the site in say, 10 minutes, the session would time out to minimise the threat of people using a computer that someone has accidentally left unlocked and unattended.

Any thoughts you have on this proposal would be greatly appreciated.

**Comment by spvickers** [comment]
You are right, this should not have happened.  But perhaps the issue is with your WebPA server as it should have expired the session after that length of inactivity.  Unless in "standby" your computer was still running the browser in the background and maintaining the connection.

**Comment by Sephster** [comment]
It may be. I will take a look into this further to see if I can find the issue and update here. Thanks @spvickers
